Jolt/Core/Semaphore.h

```diff
@@ -33,7 +34,14 @@ class JPH_EXPORT Semaphore
 	void				Acquire(uint inNumber = 1);
 
 	/// Get the current value of the semaphore
-	inline int			GetValue() const								{ return mCount; }
+	inline int			GetValue() const
+	{
+#if defined(JPH_TSAN_ENABLED) && !defined(JPH_PLATFORM_WINDOWS)
+		// TSAN complains that we're reading mCount without locking. We don't care if we read a stale value. Inserting a lock to keep TSAN happy.
```

Comment:
wouldn't it be better to use an atomic_int in relaxed mode?

Reply:
I was planning to change it to `__attribute__((no_sanitize("thread")))`, but yes that's a better idea.